.github/workflows/build.yml

```
@@ -0,0 +1,30 @@
+name: Build
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  build-amd64:
+    runs-on: ubuntu-20.04          
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+      - name: build on amd64
+        run: make modules -j$(nproc)
+  build-aarch64:
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+      - uses: uraimo/run-on-arch-action@v2
+        name: build on aarch64
```

```suggestion
        name: Build on aarch64
```

This should be changed as well, right?

---

Yes.